src/app/about/white-section/white-section.tsx

```diff
@@ -61,9 +84,9 @@ export default function WhiteSection() {
 
           <SimpleGrid
             cols={{ sm: 3, xs: 1 }}
-            justify="center"
             className="impact-facts"
-            wrap="wrap"
+            style={{justifyContent:'center',
+            wrap:"wrap"}}
```

Comment:
Simple Grid seems to take style like variables, so we can continue to utilize those

Reply:
Do you mean take it out of the style={{}} and leave it as justifyContent:'center', etc.? 
I think I changed it because after I changed to typescript it said that justifyContent wasn't a property of the simple grid

Reply:
We can revert back to `justify="center"` and `wrap="wrap"` as the component accepts those as props instead of the need for style